src/styles/katex-swap.scss

```diff
@@ -1,4 +1,7 @@
-@use 'katex' with (
+@use 'fonts.scss' with (
```

Comment:
Could you explain this change? Is there a reason not to use the whole `katex` module as before? I was hoping to keep `katex-swap.scss` as simple as possible so that it doesn't need maintenance, and just sets a compile-time variable. But I'm not an SCSS expert so I'd be happy to hear if this is a better way of doing things.